packages/spright-components/src/chat/conversation/template.ts

```diff
@@ -1,9 +1,13 @@
-import { html } from '@ni/fast-element';
+import { html, slotted } from '@ni/fast-element';
 import type { ChatConversation } from '.';
 
 /* eslint-disable @typescript-eslint/indent */
 // prettier-ignore
 export const template = html<ChatConversation>`
 <div class="messages"><slot></slot></div>
-<div class="input"><slot name="input"></slot></div>`;
+<div class="input ${x => (x.inputIsEmpty ? '' : 'has-content')}">
```

Comment:
Would be nice if there wasn't a name change + logic inversion between these, seems like unnecessary mental hoops
```suggestion
<div class="input ${x => (x.inputEmpty ? 'input-empty': '')}">
```